static/linux/webtorrent-desktop.ejs

```diff
@@ -0,0 +1,26 @@
+[Desktop Entry]
+Type=Application
+<% if (version) { %>Version=<%= version %><% } %>
+Name=<%= productName %>
+<% if (genericName) { %>GenericName=<%= genericName %><% } %>
+<% if (description) { %>Comment=<%= description %><% } %>
+Icon=<%= name %>
+<% if (name) { %>Exec=<%= name %> %U<% } %>
+Terminal=false
+Actions=CreateNewTorrent;OpenTorrentFile;OpenTorrentAddress;
+<% if (mimeType && mimeType.length) { %>MimeType=<%= mimeType.join(';') %>;<% } %>
+<% if (categories && categories.length) { %>Categories=<%= categories.join(';') %>;<% } %>
+StartupNotify=true
+<% if (name) { %>StartupWMClass=<%= name %> <% } %>
+
+[Desktop Action CreateNewTorrent]
+Name=Create New Torrent...
+<% if (name) { %>Exec=<%= name %> -n <% } %>
+
+[Desktop Action OpenTorrentFile]
+Name=Open Torrent File...
+<% if (name) { %>Exec=<%= name %> -o <% } %>
+
+[Desktop Action OpenTorrentAddress]
+Name=Open Torrent Address...
+<% if (name) { %>Exec=<%= name %> -u <% } %>
```

Comment:
These actions don't show up in Ubuntu for me when I right click on the dock icon. Is that the place to look for them? I tried installing the current latest version and I also couldn't find these, so has this just been broken for a while? Also, are you sure that `name` is the right variable? We had a path here before, I think.

Reply:
> These actions don't show up in Ubuntu for me when I right click on the dock icon. Is that the place to look for them?

I think the actions are correctly defined according to the [official documentation](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html) Furthermore, I can see them correctly in Ubuntu 18.04.3 LTS

![screenshot](https://user-images.githubusercontent.com/715354/64649458-2d42bf80-d41d-11e9-810d-9b8237ee2433.png)
 
But it would be helpful if anyone else could check this also 😅

> Also, are you sure that name is the right variable? We had a path here before, I think.

Looking in the [official documentation](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html) a `Exec` key is the one that seems we need 😉

Reply:
Okay sounds good. I was right clicking on it in the Dock, not from the launcher like you show in your screenshot. Sounds good to me.